### ISSUE TITLE
Jan 22 breaking changes

### ISSUE THREAD

**Issue by ryanrussell** [robertohuertasm/sveltekit-cognito-auth#1]
Hey Roberto,

This was a great contribution. 

Unfortunately, it looks like the sk-auth author has not been updating his repo or merging PR's lately, and the Jan 22 breaking changes to SvelteKit leave us with things like this:

```
`page` in `load` functions has been replaced by `url` and `params`
```
https://www.linkedin.com/pulse/fix-sveltekit-breaking-change-load-january-2022-tim-sonner?trk=articles_directory

Are you still using this implementation? If so, do you have plans on how to move forward?

Best,
Ryan

**Comment by robertohuertasm** [comment]
Hi @ryanrussell it seems that the author of `sk-auth` has released a new version 4 days ago (v0.4.0). I've updated the repo accordingly and it seems to be working.

**Comment by ryanrussell** [comment]
Perfect. Thanks for responding, appreciate you sharing the boilerplate!
